src/concepts/choosing_a_dataset.md

```diff
@@ -12,39 +12,40 @@ If you want to know whether a causal link occurs between two events, you can lea
 
 <!-- toc -->
 
-# Raw Pings
+## Raw Pings
 
 {{#include ../datasets/ping_intro.md}}
 
-# Main Ping Derived Datasets
+## Main Ping Derived Datasets
 
 The [main ping] includes most of the measurements that track the performance and health of Firefox in the wild. This ping includes histograms, scalars, and events.
 
 In its raw form, the main ping can be a bit difficult to work with. To make analyzing data easier, some datasets have been provided that simplify and aggregate information provided by the main ping.
 
-## `clients_daily`
+### Clients Daily
 
 {{#include ../datasets/batch_view/clients_daily/intro.md}}
 
-## `clients_last_seen`
+See the [`clients_daily` reference]( ../datasets/batch_view/clients_daily/reference.md) for more information.
 
-{{#include ../datasets/bigquery/clients_last_seen/intro.md}}
+### Clients Last Seen
 
-## `main_summary`
+{{#include ../datasets/bigquery/clients_last_seen/intro.md}}
 
-{{#include ../datasets/batch_view/main_summary/intro.md}}
+See the [`clients_last_seen` reference]( ../datasets/bigquery/clients_last_seen/reference.md) for more information.
 
-## `first_shutdown_summary`
+### Main Summary
```

Comment:
I don't think we should introduce main summary to new users, as we're actively discouraging people from using it.